build.gradle.kts

```diff
@@ -18,8 +18,8 @@ import org.jetbrains.compose.desktop.application.dsl.TargetFormat
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    kotlin("jvm") version "1.5.31"
-    id("org.jetbrains.compose") version "1.0.0"
+    kotlin("jvm") version "1.6.10"
```

Comment:
Kotlin `1.6.21` [came out a few weeks ago](https://kotlinlang.org/docs/releases.html#release-details), do we want to upgrade to that more recent version?

Reply:
The kotlin version is actually tightly coupled to the compose version, it warns you that 1.6.21 is the supported version for 1.1.1 when you build with a different kotlin version.